src/commands.c

```diff
@@ -27,6 +27,9 @@ static const CommandSpec commands[] = {
     {"save",                        CMD_SPEC_UNSUPPORTED                     },
     {"bgsave",                      CMD_SPEC_UNSUPPORTED                     },
 
+    {"eval",                        CMD_SPEC_SEND_ACL                        },
```

Comment:
This should be done also for `eval_ro` etc.

Reply:
agreed